test/IECoreScene/CurvesAlgoTest.py

```diff
@@ -32,9 +32,10 @@
 #
 ##########################################################################
 
+import IECore
```

Comment:
Is this an "IECore needs to be imported before imath" thing? As an aside, I'm thinking we should reverse our import order to match our include order (high level first, low level last), for this very reason. Any objections? Not a priority of course...